app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -63,8 +67,44 @@ describe('components/ConnectedSidebar', () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
-    wrapper.unmount();
+
+    expect(wrapper.find(`Menu`).last().prop('children')).toMatchSnapshot();
+  });
+
+  it('displays menu links for enabled Plans module', () => {
```

Comment:
@ashfahan Do we need to add this test case?. It was catered for in `Test order of menu` test case

Reply:
removed in favor of `Test order of menu`